lusSTR/format.py

```diff
@@ -15,53 +15,52 @@
 import sys
 
 
-def uas_load(input_file, sex=False):
-    '''Format a UAS Sample Details Report (.xlsx) for use with `lusSTR annotate`.'''
-    if os.path.isdir(input_file):
-        myfiles = os.listdir(input_file)
-        autosomal_data = pd.DataFrame()
-        sex_data = pd.DataFrame()
-        for filename in sorted(myfiles):
-            filepath = os.path.join(input_file, filename)
-            sex_final, results = uas_format(filepath, sex)
-            autosomal_data = autosomal_data.append(results)
-            if sex is True:
-                sex_data = sex_data.append(sex_final)
+def uas_load(inpath, sexloci=False):
+    '''Format a UAS Sample Details Report (.xlsx) for use with `lusSTR annotate`.
+
+    The `inpath` argument can refer to a report file or a directory of report files. Any files
+    without the `.xlsx` file extension are ignored. The `sexloci` argument determines whether X and
+    Y chromosome STRs are included along with autosomal STRs.
+    '''
+    if os.path.isdir(inpath):
+        files = os.listdir(inpath)
+        auto_strs = pd.DataFrame()
+        sex_strs = pd.DataFrame() if sexloci is True else None
+        for filename in sorted(files):
+            if not filename.endswith('.xlsx'):
+                continue
```

Comment:
I thought it might be a good idea to protect users from themselves, or from the operating system (i.e. those annoying `.DS_Store` files OS X makes or the `.thumbs` Windows makes).

Reply:
Great idea! The number of times I've been puzzled because my script has erred out... only to find the `.DS_Store` file....

Reply:
Ended up using `glob.glob` instead to accomplish the same thing.